app/core/src/screens/hotelsStack/placepicker/HotelCityQuery.js

```diff
@@ -1,18 +1,14 @@
 // @flow strict
 
 import * as React from 'react';
-import {
-  PublicApiRenderer,
-  graphql,
-  type GraphQLTaggedNode,
-} from '@kiwicom/mobile-relay';
+import { PublicApiRenderer, graphql } from '@kiwicom/mobile-relay';
 
 import type { HotelCityQueryResponse as HotelCityType } from './__generated__/HotelCityQuery.graphql';
 import HotelCityList from './HotelCityList';
 
 type Props = {|
   +onPress: () => void,
-  +query: GraphQLTaggedNode,
+  +query: string,
```

Comment:
Why is this change necessary?

Reply:
Because the type of query is string, and not GraphQLTaggedNode.

Reply:
It is not like the query from graphql tagget template. It is a prop called query